src/main/java/com/swordglowsblue/artifice/api/ArtificeResourcePack.java

```diff
@@ -63,6 +63,8 @@
      */
     boolean isVisible();
 
+    boolean isShouldReplace();
+
```

Comment:
Explain what this does

Reply:
It returns a Boolean value that says if it should overwrite Or not (I realize now that I used replace and overwrite interchangeably) mostly for testing purposes.

Reply:
I have updated the naming to be more consistant